### PR TITLE
Travis CI tests update - NOMKL+OPENMP, cvx NOMKL only

### PR DESCRIPTION
**Pull request by sahmed95**
The .travis.yml file in the merged changes at https://github.com/qutip/qutip/pull/873 was failing. This PR will attempt to fix this starting from the previous working version of Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ matrix:
         - python: 3.6
           env: OPENMP=1
         - python: 3.6
-          env: NOMKL=1 OPENMP=1
+          env: NOMKL=1 OPENMP=0
```
Comment (ajgpitch) [inline .travis.yml]: This makes it the same as some earlier test

**Comment by sahmed95** [comment]
Yes. I saw that. So should we have a single script for all the if statements (checking and installing python2.7, MKL, CVOPT etc.) and run it by passing some command line argument such as

```
- travis_config install_conda
- travis_config install_mkl
- travis_config install
```

to do all the complex config stuff?

**Comment by ajgpitch** [comment]
I think what you have done in the recent commit with the two if statements seems like a nice, simple, practical solution.

**Comment by ajgpitch** [comment]
@sahmed95 looks like you nailed it. 
Please merge if you are happy with it. Or mark merge ready if you want me to do it

**Comment by ajgpitch** [comment]
Ok, maybe not quite so quick.... hold on

**Comment by ajgpitch** [comment]
I notice that our mkl tests are no running with mkl version of numpy.
This has been reported by others
https://github.com/ContinuumIO/anaconda-issues/issues/9519

Solution copied here for convenience:
"please install blas=*=mkl to remedy this. Sorry for the confusion. We're working on better ways to select package variants, and we're having some growing pains."

**Comment by sahmed95** [comment]
Thanks for pointing it out but can we take one thing at a time and first merge this?

**Comment by ajgpitch** [comment]
Merged

**Comment by ajgpitch** [comment]
I will try something to ensure mkl builds for testing

**Comment by sahmed95** [comment]
Thanks for merging this.

**Pull request merged**